pyvista/utilities/reader.py

```diff
@@ -1478,29 +1530,83 @@ def set_active_time_point(self, time_point):  # noqa: D102
         self.set_active_time_value(self.time_values[time_point])
 
 
+class DICOMReader(BaseReader):
+    """DICOM Reader for reading ``.dcm`` files.
+
+    This reader reads a single file or a path containing a several ``.dcm``
+    files (DICOM stack).
+
+    Parameters
+    ----------
+    path : str
+        Path to the single DICOM (``.dcm``) file to be opened or the directory
+        containing a stack of DICOM files.
+
+    Raises
+    ------
+    FileNotFoundError
+        If ``path`` is not a DICOM file (``.dcm``) or a directory containing
+        DICOM files.
+
+    Examples
+    --------
+    >>> import pyvista
+    >>> from pyvista import examples
+    >>> path = examples.download_dicom_stack(load=False)
+    >>> reader = pyvista.DICOMReader(path)
+    >>> dataset = reader.read()
+    >>> dataset.plot(volume=True, zoom=3, show_scalar_bar=False)
+
+    """
+
+    _class_reader = _vtk.vtkDICOMImageReader
+
+    def __init__(self, path: str):
```

Comment:
I think we can get rid of this entire method now that ae03e23 gets rid of extension checks.

Reply:
I think this is now all implemented in `BaseReader` and it is redundant, but I could easily be missing something here

Reply:
You're right it's in `BaseReader` but cordoned off into a few methods that do all of it, didn't see that. I think you're right then that everything in this init method is redundant. Deleting it would resolve my complaint about the error message as well.

Reply:
Fixed in 7af391cd504166733c5eb82b4133ab2c4b0b29ef